admin_guide/cluster_management.rst

```diff
@@ -145,3 +145,24 @@ The new shard(s) are created on the same node as the shard(s) from which the ten
     102240,
     'source_host', source_port,
     'dest_host', dest_port);
+
```

Comment:
This doesn't really belong in "Cluster Management". I think we had previously discussed adding a separate section in the Admin Guide for "Table Management", which probably is a more suitable location. Could we do that?